app/config/AppConfig.scala

```diff
@@ -78,6 +79,7 @@ class AppConfigImpl @Inject() (val config: Configuration, servicesConfig: Servic
   lazy val privacy: String = config.get[String]("urls.footer.privacy")
   lazy val termsConditions: String = config.get[String]("urls.footer.termsConditions")
   lazy val govukHelp: String = config.get[String]("urls.footer.govukHelp")
+  lazy val accessibilityStatement: String = config.get[String]("urls.footer.accessibilityStatement")_
```

Comment:
Not sure the _ is required at end of the statement

Reply:
Done